scripts/src/owners/owners_file.py

```diff
@@ -61,7 +65,28 @@ def get_chart(owner_data):
     return chart
 
 
-def get_web_catalog_only(owner_data):
+def get_web_catalog_only(owner_data, raise_if_missing=False):
+    """Check the delivery method set in the OWNERS file data
+
+    Args:
+        owner_data (dict): Content of the OWNERS file. Typically this is the return value of the
+            get_owner_data or get_owner_data_from_file function.
+        raise_if_missing (bool, optional): Wether to raise an Exception if the delivery method is
+            not set in the OWNERS data. If set to False, the function returns False.
+
+    Raises:
+        ConfigKeyMissing: if the key is not found in OWNERS and raise_if_missing is set to True
+
+    """
+    if (
+        "web_catalog_only" not in owner_data
+        and "providerDelivery" not in owner_data
```

Comment:
Should these be `( ... )` grouped? Or does it matter?

Reply:
It would be the same logically to have `A & B & C` or `(A & B) & C`, and IMO would only clutter the code.